airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcSqlGenerator.java

```diff
@@ -63,7 +63,7 @@ protected DataType<?> toDialectType(final AirbyteType type) {
   protected DataType<?> toDialectType(final AirbyteProtocolType airbyteProtocolType) {
     return switch (airbyteProtocolType) {
       case STRING -> SQLDataType.VARCHAR;
-      case NUMBER -> SQLDataType.FLOAT;
+      case NUMBER -> SQLDataType.NUMERIC;
```

Comment:
DECIMAL and NUMERIC are interchangeable in redshift context. which one seems more representative for readability ?

Reply:
https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html#r_Numeric_types201-decimal-or-numeric-type

> In this document, decimal is the preferred term for this data type

I'll switch to decimal I guess :shrug: but - this is the generic jdbc generator, so maybe shouldn't index _too_ hard on redshift-specific stuff